examples/requirements.txt

```diff
@@ -1 +1,2 @@
+hydra-core>1.1
```

Comment:
@jieru-hu hydra-core is also required here, and requirements is used as part of requirements-test, so looks like a duplicate to me ?